gulpfile.js

```diff
@@ -0,0 +1,56 @@
+//
+// Code Signing
+//
+// Copyright © 2018 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Jason Leach on 2018-01-10.
+//
+
+'use strict';
+
+const gulp = require('gulp');
```

Comment:
what is this for?

Reply:
for building the app. `"build": "NODE_ENV=production gulp"`

Reply:
ahh gotchya. I'd expect that the latest version of react scripts could do the build too. I'm sure there is something extra going on here is why gulp is being used.